saeloun/miru-web
# invoice page UI changes and report page icon, UI changes

## Notion card
https://www.notion.so/6afc9500b2cc42af86be55f37894fb09?v=185bd45b9dc24dfcad9c57c07d32a7c0&p=c87dbabdf1374c5a9c3b7372f3af5a38&pm=s
https://www.notion.so/6afc9500b2cc42af86be55f37894fb09?v=185bd45b9dc24dfcad9c57c07d32a7c0&p=d964a31b05f74812905554d6ef6316b3&pm=s

## Summary

- UI changes in invoice list page
- Fix Icons for Reports

## Preview
<img width="1792" alt="Screenshot 2023-01-03 at 7 19 05 PM" src="https://user-images.githubusercontent.com/22776061/210372838-5edc1f20-1534-4160-a5dc-6f86499fb7e3.png">
<img width="1792" alt="Screenshot 2023-01-03 at 7 21 29 PM" src="https://user-images.githubusercontent.com/22776061/210372845-1db89f1d-7412-4711-a53d-6709b9bb5171.png">
<img width="1787" alt="Screenshot 2023-01-03 at 7 21 39 PM" src="https://user-images.githubusercontent.com/22776061/210372848-9a57dcce-4d33-4741-a4b9-93bb6c163a30.png">

## Type of change

- [ ] Refactor (non-breaking and retains same functionality)
      not work as expected)

## How Has This Been Tested?

- Check for UI changes in the invoice list page.
- Check for icons and UI on report page

### Checklist:

- [x] I have manually tested all workflows
- [x] I have annotated changes in the PR that are relevant to the reviewer
- [ ] I have added automated tests for my code
